docs/horizen_eon/05-forging_and_staking/05-pause_forging.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Pause Forging
+---
+
+According to the definition of Cross Chain Transfer Protocol, SC blocks contain references to MC blocks, and the SC follows the longest MC branch. This implies that if the MC undergoes a chain reorganization, the SC too will do the same, invalidating all the latest SC blocks that contain references to reverted MC blocks.

```

Comment:
Let's use `mainchain` and `sidechain` instead of `MC` and `SC`. It's more clear to the general audience.

Reply:
done